Content.Client/_Starlight/TextToSpeech/TTSSystem.cs

```diff
@@ -81,6 +83,22 @@ private void OnTtsClientOptionChanged(bool option)
     private void OnAnnounceTTSPlay(AnnounceTtsEvent ev)
         => _ttsQueue.Enqueue((ev.Data, ev.AnnouncementSound, _announceVolume));
 
+    private void OnClientTTSAudioRemove<T>(Entity<ClientTTSAudioComponent> ent, ref T args)
+    {
+        if (ent.Comp.Stream is not { } stream)
+            return;
+
+        try
+        {
+            stream.Capacity = 0;
```

Comment:
This is the only thing that feels weird to me here really

Reply:
Added a comment for this in [`14c707c` (#1031)](https://github.com/ss14Starlight/space-station-14/pull/1031/commits/14c707c468465c19d36c63eaed774691c20ee14f) after this discord conversation https://discord.com/channels/1272545509562777621/1356838103670849710/1394228381058334851